terragrunt/aws/glue/curated/platform/gc_notify/notification_enriched_test.py

```diff
@@ -0,0 +1,84 @@
+import datetime
+from dateutil.relativedelta import relativedelta
+
+
+def test_default_date_logic():
```

Comment:
Could you beef out the unit tests a bit more to cover more of the program flow.  I tend to use AI to do this 100% for me since it gives me more confidence that when I merge Renovate dependency PRs they won't break things unexpectedly (admittedly this could be false confidence).